lib/logger/lib/logger.ex

```diff
@@ -818,11 +868,20 @@ defmodule Logger do
   def __should_log__(level, module) do
     level = Logger.Handler.elixir_level_to_erlang_level(level)
 
-    if enabled?(self()) and :logger.allow(level, module) do
+    if process_allowed?(level, self()) and :logger.allow(level, module) do
```

Comment:
We can probably rely exclusively on the logger filter:

```suggestion
    if :logger.allow(level, module) do
```

Reply:
Yes. If you filter out in the filter, then it will be ignored by all handlers and all sources. Probably that would match the user expectations, but may be confusing in case if someone set `handle_otp_reports: false` (in general that single configuration option is probably the most problematic option right now).